presto-docs/src/main/sphinx/release/release-0.191.rst

```diff
@@ -0,0 +1,56 @@
+=============
+Release 0.191
+=============
+
+General Changes
+---------------
+
+* Fix regression that could cause high CPU usage for join queries when dictionary
+  processing for joins is enabled.
+* Fix :func:`bit_count` for bits between 33 and 63.
+* Add support for grouped join execution. When both sides of join
+  have the same table partitioning and the partitioning is addressable,
+  partial data can be loaded into memory at a time, making it possible to
+  execute the join with less peak memory usage. ``colocated_join`` must be
```

Comment:
This should mention both the config property and system property using the standard language (see writer scaling for an example)